plugins/lighthouse-backend/api-report.md

```diff
@@ -0,0 +1,31 @@
+## API Report File for "@backstage/plugin-lighthouse-backend"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { CatalogClient } from '@backstage/catalog-client';
+import { Config } from '@backstage/config';
+import { Logger } from 'winston';
+import { PluginTaskScheduler } from '@backstage/backend-tasks';
+
+// Warning: (ae-missing-release-tag) "create" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
To clear up these warnings you will want to annotate the export line.


```ts
/** @public **/
export const create(..)
```

Reply:
done